core/vm/evm.go

```diff
@@ -270,7 +270,7 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 			// The depth-check is already done, and precompiles handled above
 			contract := NewContract(caller, AccountRef(addrCopy), value, gas)
 			contract.SetCallCode(&addrCopy, evm.StateDB.GetCodeHash(addrCopy), code)
-			ret, err = run(evm, contract, input, false)
+			ret, err = run(evm, contract, input, false, false)
```

Comment:
I think this highlights the commens I made on FEM about the EIP: it's far too vaguely specified. In this case, it appears that executing this sequence: 

- A calls `B`. 
- `B` does `set-indesructible` as the first op. 
- `B` calls on `C`
- `C` does selfdestruct, but is prevented, because the flag was stickified from `B`.

This implementation seems based on the `static` mode, which is meant to be 'sticky' -- following along any child scopes. 
I don't interpret the EIP to have the same semantics, though